### PR TITLE
Feature Booking, SookingType, SubscriptionType entities

### PR DESCRIPTION
**Pull request by AmrEzzEldeen**
Feature Booking, BookingType, SubscriptionType entities

### CODE REVIEW

File: src/main/java/com/teamc/fms/entity/BookingType.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class BookingType {
+    @Id
+    private Integer id;
+    @NotNull
+    private String name;
+    @NotNull
+    private String description;
```
Comment (Mahmoud-Shosha) [inline src/main/java/com/teamc/fms/entity/BookingType.java]: description column is nullable
image

File: src/main/java/com/teamc/fms/entity/SubscriptionType.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class SubscriptionType {
+    @Id
+    private Integer id;
+    @NotNull
+    private String name;
+    @NotNull
+    private String description;
```
Comment (Mahmoud-Shosha) [inline src/main/java/com/teamc/fms/entity/SubscriptionType.java]: description column is nullable
image

File: src/main/java/com/teamc/fms/entity/Tenant.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    private String email;
+    @NotNull
+    private String phoneCode;
+    @NotNull
+    private String phoneNumber;
+    @NotNull
+    @ManyToOne
+    private Country nationality;
+    @NotNull
+    private LocalDate birthDate;
```
Comment (Mahmoud-Shosha) [inline src/main/java/com/teamc/fms/entity/Tenant.java:32]: phonceCode, phoneNumber, nationality and birthdate are nullable
image

**Review by Mahmoud-Shosha — CHANGES_REQUESTED** [review]
Please check these small notes

**Review by Mahmoud-Shosha — APPROVED** [review]
Approved

**Review by Muhammed9955 — APPROVED** [review]

**Pull request merged**